modules/ort/src/main/kotlin/workflow/analyzers/OrtResultAnalyzer.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.ort.workflow.analyzers
+
+import com.here.ort.model.OrtResult
+import com.here.ort.model.readValue
+
+import org.eclipse.sw360.antenna.api.exceptions.AntennaException
+import org.eclipse.sw360.antenna.api.workflow.ManualAnalyzer
+import org.eclipse.sw360.antenna.api.workflow.WorkflowStepResult
+import org.eclipse.sw360.antenna.model.artifact.Artifact
+import org.eclipse.sw360.antenna.ort.resolver.OrtResultArtifactResolver
+
+import org.slf4j.LoggerFactory
+
+import java.io.File
+import java.io.IOException
+
+class OrtResultAnalyzer : ManualAnalyzer() {
+    companion object {
+        private val LOGGER = LoggerFactory.getLogger(OrtResultAnalyzer::class.java)
+    }
+
+    init {
+        workflowStepOrder = 700
+    }
+
+    @Throws(AntennaException::class)
+    override fun yield(): WorkflowStepResult =
+        try {
+            WorkflowStepResult(createArtifactList(componentInfoFile))
+        } catch (e: IOException) {
+            throw AntennaException("Error parsing the ORT result file: ${e.message}")
```

Comment:
I propose to use the ExecutionException, which is redundant from my point of view, it is also already a runtime exception, so no throwing necessary

Reply:
Ok, I'll remove the catch / throw in a new commit. I'd like to keep it in the original porting commit to more closely resemble the original Java code in the migration step.